itests/kit/node_unmanaged.go

```diff
@@ -23,4 +34,78 @@ type TestUnmanagedMiner struct {
 		PeerID  peer.ID
 		PrivKey libp2pcrypto.PrivKey
 	}
+
+	SealedCids, UnsealedCids map[abi.SectorNumber]cid.Cid
+	SealTickets              map[abi.SectorNumber]abi.SealRandomness
+}
+
+func (tm *TestUnmanagedMiner) ManualOnboardingGeneratePreCommit(
```

Comment:
let's get rid of `ManualOnboarding`, I just prefixed with those names because utility functions in `itest/` need to be unique because they are all run as `go test ./itest/test-thing.go` so it's both a shared namespace when we dev, but a separate namespace when they are run (ugh), but that's also why it's good to get these out into the kit so they can be shared!

Reply:
Done.